script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -711,97 +786,104 @@ Do I really have to spell it out for you to get it?"
 [sha:c921bf1516b4ffea358a112a7b50820224cf0f66]{
 -- Page 469, Offset 28378.
 -- 俺は吸血鬼なんかじゃないし、ロアなんてヤツも知らない！」
-I'm not a vampire or anything like that! I don't even know this Roa guy!"
+I'm not a vampire, and I don't even know the first thing about this Roa guy!"
 }
 [sha:60cbdf4a8c823bdb822786f8240d146572fce1fa]{
 -- Page 470, Offset 28379.
 -- あまりの暴言に全身の麻痺が取れた。
-It's such a ridiculous declaration that my body shook its numbness off.
+Her accusation is so absurd that it snaps me out of my paralysis.
 }
 [sha:1422c18912fa8ed50dd125cbb950bf254a6c06cf]{
 -- Page 470, Offset 28380.
 -- だって、身に覚えがないにも程がある。
-I mean, I have absolutely no memory of having done that.
+// lol
+I mean, I feel pretty sure I'd at least %{i}notice%{/i} if I were a vampire.
 }
 [sha:1dacdb9c927d24a1e47986f611dd9eb7d3b8b947]{
 -- Page 470, Offset 28381.
 -- 俺は吸血鬼を追っている側で、間違ってもアイツらの仲間じゃない……！
-I'm on the side of those who hunt vampires, I would never ever be one of them...!
+// Buddy you are at last one vampire's 仲間
+I've been working with the people who %{i}hunt%{/i} them,
+ there's absolutely no world in which I'm on their side...!
```

Comment:
The reason it's a bit more elaborated is because that makes it sound like he's some sort of official vampire hunter guy, when he's really more just along for the ride.

Reply:
That nuance kind of exists in the JP as well, so it's not exactly that misleading to translate it that way (and I would hope most people get what he's saying anyway, but readers are very stupid sometimes)
Here's maybe a bit less of a misleading line?
>I'm on the side hunting them!